core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -116,10 +116,8 @@ public class MafUtil {
     public static final String PROTEIN_POSITION = "Protein_position";
 
     // oncotator column names
-    public static final String ONCOTATOR_DBSNP_RS = "ONCOTATOR_DBSNP_RS";
     public static final String ONCOTATOR_VARIANT_CLASSIFICATION = "ONCOTATOR_VARIANT_CLASSIFICATION";
     public static final String ONCOTATOR_REFSEQ_MRNA_ID = "ONCOTATOR_REFSEQ_MRNA_ID";
-    public static final String ONCOTATOR_UNIPROT_ENTRY_NAME = "ONCOTATOR_UNIPROT_ENTRY_NAME";
     public static final String ONCOTATOR_UNIPROT_ACCESSION = "ONCOTATOR_UNIPROT_ACCESSION";
     public static final String ONCOTATOR_CODON_CHANGE = "ONCOTATOR_CODON_CHANGE";
     public static final String ONCOTATOR_PROTEIN_POS_START = "ONCOTATOR_PROTEIN_POS_START";
```

Comment:
seems like all oncotoator_x fields can be removed?

Reply:
also below all mutation assessor references can be removed (could be separate PR but just fyi)

Reply:
mutation assessor references are removed. the public DB has values in the oncotator_xxx columns which have been renamed.